app/services/forest_liana/has_many_getter.rb

```diff
@@ -4,26 +4,61 @@ def initialize(resource, association, params)
       @resource = resource
       @association = association
       @params = params
+      @field_names_requested = field_names_requested
+    end
+
+    def field_names_requested
+      return nil unless @params[:fields] && @params[:fields][@association.table_name]
+      @params[:fields][@association.table_name].split(',')
+                                            .map { |name| name.to_sym }
     end
 
     def perform
       @records = @resource
         .unscoped
         .find(@params[:id])
         .send(@params[:association_name])
+        .select(select)
```

Comment:
The select still does not work. All the columns are retrieved :(
<img width="1231" alt="screen shot 2016-11-01 at 23 18 05" src="https://cloud.githubusercontent.com/assets/664261/19918568/984eed0a-a089-11e6-8a76-5528603433a1.png">